recipes/xraylib/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "3.1.0" %}
+
+package:
+  name: xraylib
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tschoonj/xraylib
+  git_rev: xraylib-{{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```

Comment:
@msarahan How would I know if this library is using the numpy C API?  There are numpy imports in the Cython files (import and cimport).